package.json

```diff
@@ -392,7 +392,7 @@
   "scripts": {
     "vscode:prepublish": "node node_modules/gulp/bin/gulp.js build",
     "compile": "tsc -p ./",
-    "watch": "node node_modules/gulp/bin/gulp.js initial && tsc -watch -p ./",
```

Comment:
The gulp task was used to auto-vendor editor-services first before running tsc.  Not sure why it was removed.